app/src/main/java/com/arctouch/codechallenge/view/movie/MovieItemViewModel.kt

```diff
@@ -0,0 +1,14 @@
+package com.arctouch.codechallenge.view.movie
+
+import com.arctouch.codechallenge.model.data.Movie
+
+class MovieItemViewModel(private val movie: Movie) {
```

Comment:
Do you think we could benefit from using a data class here?

Reply:
To be honest with you, I always looked the idea from data class to handle with POJOS or objects which will handle with data (i.e Retrofit) and the idea to use this to a ViewModel sounds weird to me, if you have a different idea please let me know with some article or links to take a look.

Reply:
You are right about not using a data class here since `Movie` is already one. What could be done is to either name it differently (since it's not a real view model, i. e. doesn't extend from `ViewModel` and it is not lifecycle aware), or make it lifecycle aware.

Reply:
I agree with you about name, I used ViewModel just to know I'm handling with informations, what do you suggest for a good name?

Reply:
Since we are inside an adapter, I guess `MovieItem` would be enough in this case.